projects/examples/src/components/datagrid/datagrid-row-icon.example.component.ts

```diff
@@ -0,0 +1,82 @@
+/*!
```

Comment:
Please split HTML into a separate file for syntax highlighting in the examples container